x-pack/metricbeat/module/aws/sqs/data.go

```diff
@@ -0,0 +1,40 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package sqs
+
+import (
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstrstr"
+)
+
+var (
+	schemaRequestFields = s.Schema{
+		"oldest_message_age": c.Float("ApproximateAgeOfOldestMessage"),
+		"messages": s.Object{
+			"delayed": s.Object{
+				"count": c.Int("ApproximateNumberOfMessagesDelayed"),
+			},
+			"not_visible": s.Object{
+				"count": c.Int("ApproximateNumberOfMessagesNotVisible"),
+			},
+			"visible": s.Object{
+				"count": c.Int("ApproximateNumberOfMessagesVisible"),
```

Comment:
It's such a mess to have _approximated_ values. I'm worried that users might rely on them as _source of truth_ later and will open issues wondering why their maths doesn't match what they are expecting (from the field names)

Reply:
These approximate metrics are directly from cloudwatch so if the users check in cloudwatch, the data should match what we have here.